pkg/metrics/exporter_test.go

```diff
@@ -0,0 +1,38 @@
+package metrics
+
+import (
+	"testing"
+)
+
+var (
+	metricsAddress = "8096"
+)
+
+func TestInitMetricsExporter(t *testing.T) {
+	testCases := []struct {
+		name           string
+		metricsBackend string
+		expectedError  bool
+	}{
+		{
```

Comment:
Could we also add a test to validate `strings.ToLower` in the `InitMetricsExporter`?

Reply:
Good point. Test added